docs/helm-chart/index.rst

```diff
@@ -141,6 +141,12 @@ This is so these CI/CD services can perform updates without issues and preserve
 
 This also applies if you install the chart using ``--wait`` in your ``helm install`` command.
 
+For Flux, you also need to ensure the "Jobs" are deleted and retriggered in each helmRelease, below is how you would do it:
```

Comment:
I think you need to add more information/context as to where this configuration needs to go. For instance, `spec` of which object?